modules/bemenu/testbeds/bemenu.nix

```
@@ -0,0 +1,11 @@
+{ lib, ... }:
+{
+  stylix.testbed.ui = {
+    graphicalEnvironment = "hyprland";
+    command.text = "bemenu";
+  };
+
+  home-manager.sharedModules = lib.singleton {
+    programs.bemenu.enable = true;
```

> theming doesn't seem to be applying for some reason

We need to trigger the code path that enables `home.sessionVariables` to be generated:

> ```nix
>     home.sessionVariables = lib.mkIf (cfg.settings != { }) {
>       BEMENU_OPTS = lib.cli.toGNUCommandLineShell { } cfg.settings;
>     };
> ```
>
> -- [github:nix-community/home-manager, `/modules/programs/bemenu.nix`](https://github.com/nix-community/home-manager/blob/a7b7c6f520b51f3577bd6b7a7493d2cdbcb22ec6/modules/programs/bemenu.nix#L56-L58)

I believe the following should fix it:

```suggestion
    programs = {
      bash.enable = true;
      bemenu.enable = true;
    };
```

The same is done in:

https://github.com/nix-community/stylix/blob/fc24382fabe1d4413c9a898ecbc6508c4ebb503b/modules/fzf/testbeds/fzf.nix#L9-L12

It might be better to generally `programs.bash.enable = true;` in all testbeds to avoid relying on the implementation detail of relying on environment variables. I will try to open a PR for this later.